packages/sourcecred/src/api/instance/readInstance.js

```diff
@@ -17,6 +26,16 @@ import {
   parser as credGraphParser,
 } from "../../core/credrank/credGraph";
 import {Ledger} from "../../core/ledger/ledger";
+import {
+  parser as dependenciesParser,
+  type DependenciesConfig,
+} from "../dependenciesConfig";
+import {type Budget} from "../../core/mintBudget";
+import {parser as pluginBudgetParser} from "../pluginBudgetConfig";
+import {
+  rawParser as rawConfigParser,
+  type RawInstanceConfig,
+} from "../RawinstanceConfig";
```

Comment:
ci is failing because linux is case sensitive the **"R"** needs to be small so this should be changed to 
```
} from "../rawInstanceConfig";
```

Reply:
AH! Thank you so much.